### PR TITLE
fix: Performance issue with player logs

### PR DESCRIPTION
**Pull request by benjackwhite**
## Problem

Long bit of debugging led me to find out something frustrating - the logging from the replayer is the source of a huge number of lock ups. Essentially the logged object can be so massive and full of getter proxies that simply serializing it takes such a long time that the browser locks up. 

Any recording with a lot of warnings suffered from this which is largely why some people had such bad experiences. 

## Changes

* Fixes it so we don't log, but rather just count and delay a message indicating to the viewer how they _could_ get to the error messages without actually logging them to the console
* Also accidentally includes some Notebook tidying - not too much there and I've checked it works so just going to lazily include it 🙈 

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/src/scenes/session-recordings/player/sessionRecordingPlayerLogic.ts [hunk 1]
```diff
@@ -965,13 +965,13 @@ export const sessionRecordingPlayerLogic = kea<sessionRecordingPlayerLogicType>(
...
         actions.stopAnimation()
-        cache.resetConsoleWarn?.()
+
         cache.hasInitialized = false
-        clearTimeout(cache.consoleWarnDebounceTimer)
         document.removeEventListener('fullscreenchange', cache.fullScreenListener)
         cache.pausedMediaElements = []
         values.player?.replayer?.pause()
         actions.setPlayer(null)
+        cache.unmountConsoleWarns()
```
Comment (benjackwhite) [inline frontend/src/scenes/session-recordings/player/sessionRecordingPlayerLogic.ts:974]:
```suggestion
        cache.unmountConsoleWarns?.()
```

File: frontend/src/scenes/notebooks/NotebookSelectButton/NotebookSelectButton.tsx
Comment (pauldambra) [inline frontend/src/scenes/notebooks/NotebookSelectButton/NotebookSelectButton.tsx:1]: `git checkout origin/master -- frontend/src/scenes/notebooks/NotebookSelectButton/NotebookSelectButton.tsx` is what I use to remove files from PRs when they sneak in

File: frontend/src/scenes/notebooks/Notebook/SlashCommands.tsx
Comment (pauldambra) [inline frontend/src/scenes/notebooks/Notebook/SlashCommands.tsx:1]: `git checkout origin/master -- frontend/src/scenes/notebooks/Notebook/SlashCommands.tsx` is what I use to remove files from PRs when they sneak in

**Review by pauldambra — APPROVED** [review]
additional files but 🚀

File: frontend/src/scenes/notebooks/Notebook/SlashCommands.tsx
Comment (benjackwhite) [reply frontend/src/scenes/notebooks/Notebook/SlashCommands.tsx:1]: Oh sweet. I will endeavour to do that next time 🙈

Comment (pauldambra) [reply frontend/src/scenes/notebooks/Notebook/SlashCommands.tsx:1]: I will never forgive you (it's sad, but we have to maintain standards)

**Pull request merged**